smoke/IotEdgeQuickstart/details/IBootstrapper.cs

```diff
@@ -13,7 +13,7 @@ public interface IBootstrapper
 
         Task Install();
 
-        Task Configure(DeviceProvisioningMethod method, string image, string hostname, string deviceCaCert, string deviceCaPk, string deviceCaCerts, LogLevel runtimeLogLevel);
+        Task Configure(DeviceProvisioningMethod method, string hostname, string deviceCaCert, string deviceCaPk, string deviceCaCerts, LogLevel runtimeLogLevel);
```

Comment:
Please add comments here to mention the root cause for not using provided image tag in this step.  The root cause is because edgeAgent will not be triggered a new deployment if same image tag is used; even there are environment variable changes in the edgeAgent config.  And mentioned this is a bug in edgeAgent, it will be better if you can find the corresponding github issue.

Reply:
Couldn't find a relevant issue. Do we even have this logged anywhere?